### PR TITLE
Get Sections Frontend Tentatively Done

### PR DESCRIPTION
**Pull request by kevinle350**
Some weird CSS done, not sure if thats how you want it. Left some comments on where I added some changes.

### CODE REVIEW

**Review by josephycyeh — COMMENTED** [review]
let's see, good work on the sections, let's not hard code the sections though, i'll put time on your calendar so we can talk through it, can you include screenshots or videos on the PR and future PRs

**Comment by kevinle350** [comment]
image

File: components/ItemCard.js [hunk 1]
```diff
@@ -15,138 +15,139 @@ import { Ionicons } from "@expo/vector-icons";
 import { useNavigation } from '@react-navigation/native';
```
Comment (josephycyeh) [inline components/ItemCard.js:15]: was anything really changed in this file?

**Review by josephycyeh — COMMENTED** [review]
Looks good! let me do some clean up myself  then I'll merge it in, thanks!

**Pull request merged**